.coderabbit.yaml

```
@@ -29,6 +29,10 @@ reviews:
       instructions: |
         - Check that all test cases cover edge scenarios.
         - Make sure mocking is used appropriately.
+  - paths: ["notes/views/*.py"]
+      instructions: |
```

yamllint

[warning] wrong indentation: expected 4 but found 6

(indentation)